tools/bin/check_images_exist.sh

```diff
@@ -28,7 +28,8 @@ checkNormalizationImages() {
 checkConnectorImages() {
   echo "Checking connector images exist..."
 
-  CONNECTOR_DEFINITIONS=$(grep "dockerRepository" -h -A1 airbyte-config/init/src/main/resources/seed/*.yaml | grep -v -- "^--$" | tr -d ' ')
+  # Added `: airbyte/` since we have added external images also and we don't' want to check those images
+  CONNECTOR_DEFINITIONS=$(grep "dockerRepository: airbyte/" -h -A1 airbyte-config/init/src/main/resources/seed/*.yaml | grep -v -- "^--$" | tr -d ' ')
```

Comment:
Even if it's a small change, I think it could deserve a separate PR for traceability and awareness in the Airbyte changelog.

Reply:
Yeah makes sense moved to a different PR